src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -794,6 +796,7 @@ private boolean processInstance(File instanceDir) {
       return false;
     }
     EventBus.publish(new ExportProgressEvent("Processing instance: " + instanceDir.getName()));
+    EventBus.publish(new ExportProgressPercentageEvent((++processedInstances*100.0)/totalInstances));
```

Comment:
Using the `++` operator inline like this is always a bit concerning to me. It's really easy to introduce strange off by one bugs if someone doesn't understand the difference between the prefix and suffix operator. I think it's worth adding another line to first increment the `processedInstances` variable and then calculate the percent.

Also please make sure to have spaces around all your operators (`processedInstances * 100.0` rather than processedInstances*100.0`).